aws_xray_sdk/core/models/mimic_segment.py

```diff
@@ -0,0 +1,35 @@
+from .segment import Segment
+from ..exceptions.exceptions import MimicSegmentInvalidException
+
+
+class MimicSegment(Segment):
+    """
+    The MimicSegment is an entity that mimics a segment for the use of the serverless context.
+    When the MimicSegment is generated, its parent segment is assigned to be the FacadeSegment
+    generated by the Lambda Environment. Upon serialization and transmission of the MimicSegment,
+    it is converted to a locally-namespaced, subsegment. This is only done during serialization.
+    All Segment-related method calls done on this object are valid.
+
+    Subsegments are automatically created with the namespace "local" to prevent it from appearing
+    as a node on the service graph. For all purposes, the MimicSegment can be interacted as if it's
+    a real segment, meaning that all methods that exist only in a Segment but not a subsegment
+    is available to be used.
+    """
+
+    def __init__(self, facade_segment=None, original_segment=None):
```

Comment:
If these arguments are all required, don't give them default. The default `None` should only be given when the argument is optional.

Reply:
Good point